apps/src/radar.rs

```diff
@@ -315,19 +356,32 @@ fn main() {
                         let table = Table::new(rows)
                             .style(Style::default().fg(Color::White))
                             .header(
-                                Row::new(vec!["ICAO Address", "Longitude", "Latitude", "Altitude"])
-                                    .bottom_margin(1),
+                                Row::new(vec![
+                                    "ICAO",
+                                    "Call sign",
+                                    "Longitude",
+                                    "Latitude",
+                                    "Altitude",
+                                    "   FPM",
```

Comment:
Maybe there's a better way to do the label justification in TUI?

Reply:
I don't see any location to set this in the `tui::widgets::Cell`